.travis.yml

```diff
@@ -15,31 +15,60 @@ addons:
       - google-chrome-stable
 cache:
   directories:
-    - $HOME/.m2
-    - $HOME/.gradle
+    - "$HOME/.m2"
+    - "$HOME/.gradle"
 env:
   global:
+    # If you add some global variables, please add it also at the end of ./travis/build-samples.sh - PROFILE=dev
     - PROFILE=dev
-    - RUN_APP=1
     - PROTRACTOR=0
-    - JHIPSTER_TRAVIS=$TRAVIS_BUILD_DIR/travis
-    - JHIPSTER_SAMPLES=$JHIPSTER_TRAVIS/samples
-    - JHIPSTER_SCRIPTS=$JHIPSTER_TRAVIS/scripts
-    - APP_FOLDER=$HOME/app
-    - UAA_APP_FOLDER=$HOME/uaa
+    - JHIPSTER_TRAVIS="$TRAVIS_BUILD_DIR/travis"
```

Comment:
@pascalgrimaud hope you will review changes to this file

Reply:
@deepu105 : yes, I will review this PR as soon as it's finished